.gitignore

```diff
@@ -41,3 +41,4 @@ website/public/
 
 # Don't ignore directory of Github workflows
 !.github
+!.pre-commit-config.yaml
```

Comment:
Why do you add the pre-commit config.yaml to the gitignore file? AFAIK this file should be tracked?

Reply:
Sorry for my misunderstanding, didn't catch that earlier on😅 Thanks for the explanation!

Reply:
No problem - critically questioning my contributions is always a good thing :wink:

Reply:
FYI, even if you exclude all hidden files with `.*`, you can add them to the repo with `git add -f .pre-commit-config.yaml` then it will be tracked even tough it is excluded in `.gitignore`, that's also how the `.gitignore` file is tracked.